frontend/src/components/Toolbar/Toolbar.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Box from "@material-ui/core/Box";
```

Comment:
Note: there is an empty file https://github.com/VerWiki/VerWikiApp/pull/9/files#diff-ac65a74bed66a760c169a3c4308bbb86e1ed69ad486616886083b663531a3583

Reply:
Yup ik. Right now, I don't see the need to put any styles there, but that's where any future CSS styling would go. I also decided to keep it to remain consistent with the directory structure.